src/Stratis.Bitcoin/Utilities/JsonConverters/NetworkJsonConverter.cs

```diff
@@ -1,48 +1,65 @@
 ﻿using System;
 using System.Reflection;
+using NBitcoin;
 using Newtonsoft.Json;
 
-namespace NBitcoin.JsonConverters
+namespace Stratis.Bitcoin.Utilities.JsonConverters
 {
+    /// <summary>
+    /// Converter used to convert the name of a network in JSON to the corresponding <see cref="Network"/>.
+    /// </summary>
+    /// <seealso cref="Newtonsoft.Json.JsonConverter" />
     public class NetworkJsonConverter : JsonConverter
     {
+        /// <inheritdoc />
         public override bool CanConvert(Type objectType)
         {
             return typeof(Network).GetTypeInfo().IsAssignableFrom(objectType.GetTypeInfo());
         }
 
+        /// <inheritdoc />
         public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
         {
             if (reader.TokenType == JsonToken.Null)
                 return null;
 
             string network = (string)reader.Value;
+
             if (network == null)
                 return null;
+
             if (network.Equals("MainNet", StringComparison.OrdinalIgnoreCase) || network.Equals("main", StringComparison.OrdinalIgnoreCase))
                 return Network.Main;
+
             if (network.Equals("TestNet", StringComparison.OrdinalIgnoreCase) || network.Equals("test", StringComparison.OrdinalIgnoreCase))
                 return Network.TestNet;
-            if(network.Equals("RegTest", StringComparison.OrdinalIgnoreCase) || network.Equals("reg", StringComparison.OrdinalIgnoreCase))
+
+            if (network.Equals("RegTest", StringComparison.OrdinalIgnoreCase) || network.Equals("reg", StringComparison.OrdinalIgnoreCase))
                 return Network.RegTest;
+
             Network net = NetworksContainer.GetNetwork(network);
```

Comment:
Might as well rename to `network` here too,

Reply:
there is already a network object in the class. So I've renamed `network` to `networkName` and `net` to `network`.